src/components/Book.js

```diff
@@ -0,0 +1,8 @@
+const Book = () => (
+  <div className="book-container">
+    <span className="book-title">Book title</span>
+    <img src="https://img.icons8.com/ios/20/trash--v1.png" alt="trash--v1" />
+  </div>
+);
```

Comment:
- [ ] As I mentioned above :point_up_2: pass `title` and `author` props inside the `Book` function then you will able to use these props in your `Book` component and use these props in your `spam` tag :slightly_smiling_face: 
- [ ] Please remove the `img` tag and replace it with the `button` tag with `Remove` text. :slightly_smiling_face: 

**Check below :point_down: my suggestions**
```suggestion
const Book = ({title, author}) => (
  <div className="book-container">
    <span className="book-title">{title}</span>
    <span className="book-title">{author}</span>
    <button>Remove</button>
  </div>
);
```